react-native-libraries.json

```
@@ -12405,5 +12405,13 @@
     ],
     "images": ["https://raw.githubusercontent.com/EdgarJMesquita/expo-pip/refs/heads/main/docs/assets/demo.gif"],
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/tamagui/tamagui",
```

Hello @natew, thanks for adding Tamagui to the directory! 👍 

In monorepos, the `githubUrl` need to point to the directory which includes `package.json` for a given package:

```suggestion
    "githubUrl": "https://github.com/tamagui/tamagui/tree/master/code/ui/tamagui",
```

Let's update that to fix the CI and unblock shippment.

---

Done, thanks!